common/src/web_api.rs

```diff
@@ -24,7 +24,7 @@ pub struct CreateAccountOptions {
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase")]
 pub struct UploadSignedPreKey {
-    pub key_id: u32,
+    pub key_id: u64,
```

Comment:
ensure the client can handle a u64 as the id

Reply:
This is defined by by the specification therefore it needs to be 8 bytes long.

Reply:
Found out that libsignal doesnt even support using an 8 byte variant. Therefore i just added a comment such that we are aware of this difference that our design could have compared to theirs.